PicPi/HashableAdapter.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  HashableAdapter.swift
+//  PicPi
+//
+//  Created by Erick Sanchez on 12/24/20.
+//
+
+import Foundation
+
+/// Wrap a non hashable type and hash via the given id.
+struct HashableAdapter<NonHashable> {
+  let id: String
```

Comment:
what is the "id" based on?
it should be unique for each pictureFrame, right?

Reply:
closing this since it's discussed in a later comment